.github/workflows/publishbot.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Robot
+on:
+  push:
+    branches:
+      - master
+    tags: ['*']
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+  npm-publish:
+    name: 'Publish to NPM'
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+      - run: npm run npm:publish:alpha
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  deploy:
+    name: 'Deploy the package'
+    needs: build
```

Comment:
Should we only deploy if we publish successfully? Should we only publish if we deploy successfully? It looks like in Jenkins we did both regardless, and I don't see much of a reason not to do that here either. Both of these jobs run in parallel.

Reply:
We could make a successful publish a prerequisite to creating a deployment package, since the qa and prd steps will be trying to adjust the tag of the package.